tests/ServingTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Copyright 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\tests;
+
+use GuzzleHttp\Client;
+
+class ServingTest extends \PHPUnit_Framework_TestCase
+{
+
+    private $pid;
+    private $client;
+
+    public function setUp()
+    {
+        exec('docker run -d --name php56 -p 127.0.0.1:8080:8080 php56');
+        // Wait for nginx to start
+        sleep(3);
```

Comment:
Theoretically yes, but it works well for now. If it works reliably, I don't think it's warrant to spend engineering time. Can I revisit when the tests become flaky?